src/main/java/org/kuali/kfs/sys/document/authorization/AccountingDocumentAuthorizerBase.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * The Kuali Financial System, a comprehensive financial management system for higher education.
+ *
+ * Copyright 2005-2021 Kuali, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.kuali.kfs.sys.document.authorization;
+
+import org.kuali.kfs.coa.businessobject.Account;
+import org.kuali.kfs.coa.service.AccountService;
+import org.kuali.kfs.krad.document.Document;
+import org.kuali.kfs.sys.businessobject.AccountingLine;
+import org.kuali.kfs.sys.businessobject.FinancialSystemDocumentHeader;
+import org.kuali.kfs.sys.identity.KfsKimAttributes;
+import org.kuali.kfs.kim.api.identity.Person;
+
+import java.util.Map;
+
+/**
+ * DocumentAuthorizer containing common, reusable document-level authorization code for financial (i.e. Transactional)
+ * documents
+ */
+/**
+ * 
+ * CU customization: Backport FINP-9249. The backport has been added to a copy of this file is from the 28-01-2021 version of financials.
```

Comment:
Should this comment reference FINP-8249 instead of FINP-9249?

Reply:
Good catch. I am updating it.